src/components/notifications/Slack.vue

```diff
@@ -20,6 +20,9 @@
             <i18n-t tag="p" keypath="aboutWebhooks" style="margin-top: 8px;">
                 <a href="https://api.slack.com/messaging/webhooks" target="_blank">https://api.slack.com/messaging/webhooks</a>
             </i18n-t>
+            <p style="margin-top: 8px;">
+                {{ $t("aboutUsername", [$t("slack"), $t("Username"), $t("Friendly Name")]) }}
```

Comment:
also, lets move this up to where the username field is, so that it is not so out of place